lib/configuration/configuration.ts

```diff
@@ -53,33 +60,35 @@ export const getConfiguration = async (): Promise<void> => {
     const res = await axiosInstance.get<Configuration[]>(
       `${config.configurationApiUrlV1}/api/v1/configuration?types=MMH`,
     );
-    res.data.forEach(({ type, featureToggles }) => {
-      const toggles = featureToggleStore.getValue();
-      featureToggleStore.next({
-        ...toggles,
+    res.data.forEach(({ type, featureToggles, configuration }) => {
+      const configs = configurationStore.getValue();
+      configurationStore.next({
+        ...configs,
         [type]: {
           ...featureToggles,
+          ...configuration,
```

Comment:
You've defined the configuration property as `AppConfigs` but here you specify `configuration`. Which is correct? I prefer the later.

Reply:
Reverted back to `configuration` from `AppConfig`